### PR TITLE
LPS-64866 Kaleo Workflow upgrade failing on GroupImpl

### PR DESCRIPTION
**Pull request by rafaprax**

### CODE REVIEW

**Comment by marcellustavares** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

File: modules/apps/forms-and-workflow/portal-workflow/portal-workflow-kaleo-service/src/main/java/com/liferay/portal/workflow/kaleo/upgrade/v1_3_0/UpgradeClassNames.java [hunk 1]
```diff
@@ -63,7 +63,9 @@ protected String renamePortalJavaClassNames(String workflowContextJSON) {
 		while (matcher.find()) {
 			String oldPortalJavaClassName = matcher.group(1);
 
-			if (oldSubs.contains(oldPortalJavaClassName)) {
+			if (!oldPortalJavaClassName.contains(".impl") ||
```
Comment (marcellustavares) [inline modules/apps/forms-and-workflow/portal-workflow/portal-workflow-kaleo-service/src/main/java/com/liferay/portal/workflow/kaleo/upgrade/v1_3_0/UpgradeClassNames.java]: @rafaprax, this is really a negation? I didn't get it

Comment (rafaprax) [inline modules/apps/forms-and-workflow/portal-workflow/portal-workflow-kaleo-service/src/main/java/com/liferay/portal/workflow/kaleo/upgrade/v1_3_0/UpgradeClassNames.java]: @marcellustavares, Sorry, my mistake. Fixed and rebased

**Comment by marcellustavares** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Comment by marcellustavares** [comment]
Pull request submitted to https://github.com/brianchandotcom/liferay-portal/pull/38576. <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request closed without merging**